clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,443 @@
+	req := c.LegacyAuthorizationsApi.GetAuthorizations(ctx)
+
+	if params.User != "" {
+		req = req.User(params.User)
+	}
+	if params.UserID != "" {
+		req = req.UserID(params.UserID)
+	}
+	if params.OrgID.Valid() {
+		req = req.OrgID(params.OrgID.String())
+	}
+	if params.OrgName != "" {
+		req = req.Org(params.OrgName)
+	}
+	if params.Username != "" {
+		req = req.Token(params.Username)
+	}
+	if params.ID.Valid() {
+		req = req.AuthID(params.ID.String())
+	}
+
+	auths, err := req.Execute()
+	if err != nil {
+		return fmt.Errorf("could not find Authorizations for specified arguments %w", err)
+	}
+
+	var tokens []v1Token
+	for _, a := range auths.GetAuthorizations() {
+		var permissions []string
+		for _, p := range a.GetPermissions() {
+			permissions = append(permissions, permString(p))
+		}
+
+		usr, err := c.UsersApi.GetUsersID(ctx, a.GetUserID()).Execute()
+		if err != nil {
+			return fmt.Errorf("could not find user with ID %q: %w", a.GetUserID(), err)
+		}
+
+		tokens = append(tokens, v1Token{
+			ID:          a.GetId(),
+			Description: a.GetDescription(),
+			Token:       a.GetToken(),
+			Status:      a.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: permissions,
+		})
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{tokens: tokens})
+}
+
+type ActiveParams struct {
+	AuthLookupParams
+}
+
+func (c Client) SetActive(ctx context.Context, params *ActiveParams, active bool) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	req := c.LegacyAuthorizationsApi.PatchAuthorizationsID(ctx, id)
+	var status string
+	if active {
+		status = "active"
+	} else {
+		status = "inactive"
+	}
+	req.AuthorizationUpdateRequest(api.AuthorizationUpdateRequest{
+		Status: &status,
+	})
+
+	auth, err := req.Execute()
+	if err != nil {
+		return fmt.Errorf("failed to update status of authorization with ID %q: %w", id, err)
+	}
+
+	usr, err := c.UsersApi.GetUsersID(ctx, auth.GetUserID()).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find user from User ID %q: %w", auth.GetUserID(), err)
+	}
+
+	ps := make([]string, 0, len(auth.GetPermissions()))
+	for _, p := range auth.GetPermissions() {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type SetPasswordParams struct {
+	AuthLookupParams
+	Password string
+}
+
+func (c Client) SetPassword(ctx context.Context, params *SetPasswordParams) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	auth, err := c.LegacyAuthorizationsApi.GetAuthorizationsID(ctx, id).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find authorization with User ID %q: %w", id, err)
+	}
```

Comment:
I think we can get rid of this block. It looks like the only place `auth` is used below is to get the ID, and we get that from `getAuthReqID` above.